components/brave_wallet_ui/components/extension/connection/connection.tsx

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+// Types
+import { BraveWallet } from '../../../constants/types'
+
+// Queries
+import { useGetActiveOriginQuery } from '../../../common/slices/api.slice'
+
+// Utils
+import { getLocale } from '$web-common/locale'
+
+// Components
+import {
+  WalletPageWrapper //
+} from '../../desktop/wallet-page-wrapper/wallet-page-wrapper'
+import {
+  DefaultPanelHeader //
+} from '../../desktop/card-headers/default-panel-header'
+import { CreateSiteOrigin } from '../../shared/create-site-origin/index'
+import {
+  ConnectionSection //
+} from './components/connection_section/connection_section'
+
+// Styled Components
+import { FavIcon } from './connection.style'
+import { Column, Text } from '../../shared/style'
+
+const CONNECTABLE_COIN_TYPES = [
+  BraveWallet.CoinType.ETH,
+  BraveWallet.CoinType.SOL
+]
+
+export const Connection = () => {
+  // Queries
+  const { data: activeOrigin = { eTldPlusOne: '', originSpec: '' } } =
+    useGetActiveOriginQuery()
+
+  return (
+    <WalletPageWrapper
+      wrapContentInBox={true}
+      noCardPadding={true}
+      useDarkBackground={true}
+      isConnection={true}
+      cardHeader={
+        <DefaultPanelHeader title={getLocale('braveWalletConnection')} />
+      }
+    >
+      <Column
+        fullWidth={true}
+        padding='8px 16px'
+      >
+        <FavIcon
+          src={`chrome://favicon/size/64@1x/${activeOrigin.originSpec}`}
```

Comment:
we should use chrome://favicon2 here instead and then URL encode it. chrome://favicon is deprecated now

Reply:
I have fixed this to now use `chrome://favicon2` and `encodeURIComponent`. Also noticed other parts of the wallet were not using this, those have now been updated as well.

Reply:
Changes here https://github.com/brave/brave-core/compare/3c2f962cf450fd06d232852985c0fa256efdd344..5fb81f31ed0fd2aedcd4dcbc151c6f2028fd7a17